lib/runtime/Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "wasmer-runtime"
 version = "0.13.1"
 description = "Wasmer runtime library"
-documentation = "https://wasmerio.github.io/wasmer/c/runtime-c-api/"
```

Comment:
This was a wrong doc link, intended to be the `runtime-c-api` not for the `runtime`.

We are fixing it in this PR :)